python/lsst/obs/base/cameraMapper.py

```diff
@@ -789,10 +792,14 @@ def _setupRegistry(self, name, path, policy, policyKey, storage, searchParents=T
                 if newPath is not None:
                     path = newPath
             self.log.debug("Loading %s registry from %s", name, path)
-            registry = dafPersist.Registry.create(storage.getLocalFile(path))
+            localFileObj = storage.getLocalFile(path)
+            registry = dafPersist.Registry.create(localFileObj.name)
```

Comment:
I presume it's OK that `localFileObj` can go out of scope and delete temporary file associated with it?